data/strings/strings.txt

```diff
@@ -21965,86 +21966,91 @@
     zh-Hant = 避開收費公路
 
   [avoid_unpaved]
+    comment = Recommended length for CarPlay and Android Auto is around 25-27 characters
     tags = android,ios
     en = Avoid unpaved roads
-    ar = تجنب الطرق الترابية
+    af = ermy grondpaaie
```

Comment:
Ermy?

Reply:
I think copypasting somewhere the "V" was lost. Correct translation seems to be "Vermy grondpaaie". Fixed